examples/common/scripts/vttablet-up.sh

```diff
@@ -56,6 +56,7 @@ vttablet \
  --heartbeat_enable \
  --heartbeat_interval=250ms \
  --heartbeat_on_demand_duration=5s \
+ --vreplication_experimental_flags=7 \
```

Comment:
I will remove this prior to merging. I was using it with local tests to speed things up.